src/release.js

```diff
@@ -188,7 +200,7 @@ module.exports = function taoExtensionReleaseFactory(params = {}) {
             // Start with CLI option, if it's missing we'll prompt user
             let comment = releaseComment;
 
-            if (!comment || !comment.length) {
+            if (interactive && (!comment || !comment.length)) {
```

Comment:
I'll set it to an empty string, because the non-interactive mode can also be used _manually_. And we can add the comment `Automated release` as a parameter in the ci